src/lib/containers/widgets/facet-nav/SelectionCriteriaPills.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react'
+import { QueryContextType, useQueryContext } from '../../QueryContext'
+import {
+  ColumnMultiValueFunctionQueryFilter,
+  ColumnSingleValueQueryFilter,
+  isColumnMultiValueFunctionQueryFilter,
+  isColumnSingleValueQueryFilter,
+  isTextMatchesQueryFilter,
+  QueryFilter,
+  TextMatchesQueryFilter,
+} from '../../../utils/synapseTypes/Table/QueryFilter'
+import SelectionCriteriaPill, {
+  SelectionCriteriaPillProps,
+} from './SelectionCriteriaPill'
+import {
+  FacetColumnRequest,
+  isFacetColumnRangeRequest,
+  isFacetColumnValuesRequest,
+} from '../../../utils/synapseTypes'
+import {
+  QueryVisualizationContextType,
+  useQueryVisualizationContext,
+} from '../../QueryVisualizationWrapper'
+
+function getPillPropsFromColumnQueryFilter(
```

Comment:
An individual QueryFilter/FacetColumnRequest may have multiple values, which means multiple pills should be shown. These functions control that. Hopefully much more clean than what we had in `TotalQueryResults` before this PR.